jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/sftp/AutoConfiguration.java

```diff
@@ -42,8 +42,8 @@
     public AutoConfiguration(SftpInputProperties sftpInputProperties) {
         this.properties = sftpInputProperties;
 
-        logger.debug("SFTP Configuration: Host => [{}]", this.properties.getHost());
-        logger.debug("SFTP Configuration: Port => [{}]", this.properties.getPort());
+        logger.debug("SFTP Configuration:\n");
+        logger.debug(this.properties.formatPublicFields());
```

Comment:
Agree there should something that logs all the property but it should be 1 log per property same idea, easier to parse

Reply:
Sounds good to me!